CoreOpenTibiaServer/src/COTS.Infra.CrossCutting.Ioc/BootStrapper.cs

```diff
@@ -22,18 +24,43 @@ public static void ConfigureGlobalServices(IServiceCollection serviceCollection)
             serviceCollection.AddLogging();
             serviceCollection.AddOptions();
 
-            // add services 
-            serviceCollection.AddTransient<IPlayerService, PlayerService>();
-            serviceCollection.AddTransient<IPlayerRepository, PlayerRepository>();
+            // All Repositories
+            var repositoryAssembly = typeof(AccountRepository).GetTypeInfo().Assembly;
```

Comment:
I know that you guys will say that I'm using reflection `GetTypeInfo()` to do that trick, but as it is executed only one time before the applications runs, and no more when it is running, there is no performance problem here and this code is safe.